yarpcerrors/errors.go

```diff
@@ -159,227 +159,163 @@ func ErrorMessage(err error) string {
 }
 
-// IsResourceExhausted returns true if ErrorCode(err) == CodeResourceExhausted.
-//
-// Deprecated: Use FromError and Code instead.
+// IsResourceExhausted returns true if Status(err).Code() == CodeResourceExhausted.
 func IsResourceExhausted(err error) bool {
-	return ErrorCode(err) == CodeResourceExhausted
+	return Status(err).Code() == CodeResourceExhausted
 }
 
-// IsFailedPrecondition returns true if ErrorCode(err) == CodeFailedPrecondition.
-//
-// Deprecated: Use FromError and Code instead.
+// IsFailedPrecondition returns true if Status(err).Code() == CodeFailedPrecondition.
 func IsFailedPrecondition(err error) bool {
-	return ErrorCode(err) == CodeFailedPrecondition
+	return Status(err).Code() == CodeFailedPrecondition
 }
 
-// IsAborted returns true if ErrorCode(err) == CodeAborted.
-//
-// Deprecated: Use FromError and Code instead.
+// IsAborted returns true if Status(err).Code() == CodeAborted.
 func IsAborted(err error) bool {
-	return ErrorCode(err) == CodeAborted
+	return Status(err).Code() == CodeAborted
 }
 
-// IsOutOfRange returns true if ErrorCode(err) == CodeOutOfRange.
-//
-// Deprecated: Use FromError and Code instead.
+// IsOutOfRange returns true if Status(err).Code() == CodeOutOfRange.
 func IsOutOfRange(err error) bool {
-	return ErrorCode(err) == CodeOutOfRange
+	return Status(err).Code() == CodeOutOfRange
 }
 
-// IsUnimplemented returns true if ErrorCode(err) == CodeUnimplemented.
-//
-// Deprecated: Use FromError and Code instead.
+// IsUnimplemented returns true if Status(err).Code() == CodeUnimplemented.
 func IsUnimplemented(err error) bool {
-	return ErrorCode(err) == CodeUnimplemented
+	return Status(err).Code() == CodeUnimplemented
 }
 
-// IsInternal returns true if ErrorCode(err) == CodeInternal.
-//
-// Deprecated: Use FromError and Code instead.
+// IsInternal returns true if Status(err).Code() == CodeInternal.
 func IsInternal(err error) bool {
-	return ErrorCode(err) == CodeInternal
+	return Status(err).Code() == CodeInternal
 }
 
-// IsUnavailable returns true if ErrorCode(err) == CodeUnavailable.
-//
-// Deprecated: Use FromError and Code instead.
+// IsUnavailable returns true if Status(err).Code() == CodeUnavailable.
 func IsUnavailable(err error) bool {
-	return ErrorCode(err) == CodeUnavailable
+	return Status(err).Code() == CodeUnavailable
 }
 
-// IsDataLoss returns true if ErrorCode(err) == CodeDataLoss.
-//
-// Deprecated: Use FromError and Code instead.
+// IsDataLoss returns true if Status(err).Code() == CodeDataLoss.
 func IsDataLoss(err error) bool {
-	return ErrorCode(err) == CodeDataLoss
+	return Status(err).Code() == CodeDataLoss
 }
 
-// IsUnauthenticated returns true if ErrorCode(err) == CodeUnauthenticated.
-//
-// Deprecated: Use FromError and Code instead.
+// IsUnauthenticated returns true if Status(err).Code() == CodeUnauthenticated.
 func IsUnauthenticated(err error) bool {
-	return ErrorCode(err) == CodeUnauthenticated
+	return Status(err).Code() == CodeUnauthenticated
```

Comment:
`Status` is the struct type, this is a cast, I think what you meant was `FromError(err).Code()`